lib/pushy_client/protocol_handler.rb

```diff
@@ -476,6 +468,35 @@ def send_signed_json_command(method, json)
       end
     end
 
+    # This method validate and generate the message
+    # params user packet
+    # Return Waring or JSON message
```

Comment:
This still looks like not fixed. This should be "Returns warning or regenerated JSON message"